### DISCUSSION TITLE
I can not find out where is code to remove old data in dataframe.

### DISCUSSION THREAD

**Discussion by ChainCoderX** [rodrigo-brito/ninjabot#175] [Q&A] [answered]
I can not find out where is code to remove old data in the data frame.

Could you please share the code that removes the old data in the data frame?

**Comment by rodrigo-brito** [comment, marked as answer]
In the current version, It will be cleaned only after a application restart.

**Comment by ChainCoderX** [comment]
got it.

thanks a lot.
